DFC.App.DiscoverSkillsCareers/Views/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -1,2 +1,3 @@
-﻿<script src="~/lib/jquery-validation/dist/jquery.validate.min.js"></script>
+﻿<script src="~/lib/jquery/dist/jquery.js"></script>
+<script src="~/lib/jquery-validation/dist/jquery.validate.min.js"></script>
```

Comment:
Do not use local JS, should be using CDN (dfc-assets)

Reply:
Modified to use cdn